en/core-libraries/validation.rst

```diff
@@ -103,16 +102,15 @@ The ``Table`` class provides a validation rule to ensure that a given field
 is unique within a table. For example, if you wanted to make sure that an e-mail
 address is unique, you could do the following::
 
-    $validator->add('email', [
-        'unique' => ['rule' => 'validateUnique', 'provider' => 'table']
+    $validator->add('email', 'unique', [
+        'rule' => 'validateUnique', 'provider' => 'table']
```

Comment:
it was also correct before, if you flatten this you also have to remove the extra closing ] then :)